### DISCUSSION TITLE
Normal distribution for morpho

### DISCUSSION THREAD

**Discussion by asmaaharfoush** [neperfepx/neper#179] [Q&A] [answered]
Hello, I am wondering why there is no normal distribution in morpho. Only lognormal is an option. Is there any way to create a normal distribution for the grain size?

I only can do this "neper -T -n from_morpho -morpho 'diameq:lognormal(0.1,0.42),1-circularity:lognormal(0.100,0.03)' -o 0 -dim 2"

Also, what is the dimension of the mean and standard deviation if I am going to use the lognormal, is it in micro mm?

Thank you!

**Comment by rquey** [comment, marked as answer]
Just change `lognormal` into `normal` and it will work... The full list of available distributions is provided in the documentation.

If you provide a number of grains (which is not the case here), the mean and standard deviation are considered _relative_.
If you use `-n from_morpho` (which is the case here), the mean and standard deviation are considered _absolute_, but they are also unitless. The can be meters if you want. Just be aware that the domain size is 1x1x1 by default.

**Reply by asmaaharfoush** [reply]
Yes, this works, thank you!
